Update README.md (#11)
itsmeow/Spicetify-Canvas: README.md

Create a batch script inside `C:\code\`, name it `build.bat`. If the name change doesn't change the icon, press View in file explorer and check "File name extensions" and rename it again. Paste the following into this script and save it (you can open it by right clicking and pressing edit):

> Spotify recently switched to 64-bit, you can check this in your spotify app by clicking on the three menu dots in the top left, help, about Spotify. The following intsructions will continue wtih the assumption that you want to build cef for 64-bit.

Reviewer: ```suggestion
Spotify recently switched to 64-bit, you can check this in your Spotify app by clicking on the three menu dots in the top left, then Help -> About Spotify. The following instructions will continue with the assumption that you want to build CEF for 64-bit.
```

```
set CEF_USE_GN=1

> set GN_DEFINES=is_official_build=true is_component_build=true proprietary_codecs=true ffmpeg_branding=Chrome use_thin_lto=false target_cpu ="x64" enable_nacl=false blink_symbol_level=0 symbol_level=0

Reviewer: ```suggestion
set GN_DEFINES=is_official_build=true is_component_build=true proprietary_codecs=true ffmpeg_branding=Chrome use_thin_lto=false target_cpu="x64" enable_nacl=false blink_symbol_level=0 symbol_level=0
```

- resources.pak
- chrome_100_percent.pak
- chrome_200_percent.pak

> - cef_extensions.pak EDIT: Does not exist anymore

Reviewer: ```suggestion
```

- The entire `locales` folder

Next, from the `Release_GN_x64` folder (or `binary_distrib`), copy all of the files and folders with names matching the above into your Spotify folder.

The `locales` folder you copy will contain a bunch of `.info`, `.pdb`, and `.lib` files. You can delete any files that are not `.pak` or `.dll`.

Note that Spotify includes a set of `.mo` files in its locales folder. Do not delete them.

You should also copy the same set of files you copied into Spotify into another, safe folder, in case Spotify updates or is uninstalled. You DO NOT want to lose them and rebuild Chromium. I also copied my entire `Release_GN_x64` folder just to be sure, but it may be large.

Finally, start Spotify! If you did things correctly, Spotify should run as normal, except now the Canvases will appear. You can delete the contents of `C:/code/chromium_git` and `C:/code/depot_tools` to save space, but I recommend keeping the script just in case Spotify updates.

If you made it this far, you are truly technically competent! Congratulations.

Now that you have shown that you are truly competent, here is a little lazy upgrade script that you can use inside your build.bat:

> ```

Reviewer: ```suggestion
```batch
```

echo %output%

set CEF_USE_GN=1

> set GN_DEFINES=is_official_build=true is_component_build=true proprietary_codecs=true ffmpeg_branding=Chrome use_thin_lto=false target_cpu ="x64" enable_nacl=false blink_symbol_level=0 symbol_level=0

Reviewer: ```suggestion
set GN_DEFINES=is_official_build=true is_component_build=true proprietary_codecs=true ffmpeg_branding=Chrome use_thin_lto=false target_cpu="x64" enable_nacl=false blink_symbol_level=0 symbol_level=0
```
